### PR TITLE
Remove BOM from gc.asm

### PR DESCRIPTION
**Pull request by PontusPih**
Palbart assumes input is 7-bit ascii and will not accept the Byte Order
Mark added to the start of UTF-8 files by certain IDEs.

### CODE REVIEW

File: Emulator/pal/gc.asm [hunk 1]
```diff
@@ -10,15 +10,13 @@ TCHR,	0000					/TEMP CHAR
 
 		*200					/LOCATE @ 0200
 
-		CLA						/CLEAR ACC
-		CLL						/CLEAR LINK
+		CLA CLL					/CLEAR ACC, CLEAR LINK
```
Comment (PontusPih) [inline Emulator/pal/gc.asm]: palbart will combine microcoded instructions if you write them on the same line.

File: Emulator/pal/gc.asm [hunk 2]
```diff
@@ -10,15 +10,13 @@ TCHR,	0000					/TEMP CHAR
...
 		TAD MSG					/LOAD ADDR OF MESSAGE
 		DCA AIX1				/DUMP IN AUTOINDEX REGISTER #1
 
-		CLA						/CLEAR ACC
-		CLL						/CLEAR LINK
+		CLA CLL					/CLEAR ACC, CLEAR LINK
 
-LOOP,	NOP						/READ AND PRINT LOOP
+LOOP,							/READ AND PRINT LOOP
```
Comment (PontusPih) [inline Emulator/pal/gc.asm]: NOP is an unnecessary instruction here. Sins you don't call LOOP with JMS there is no need to reserve space. The label "LOOP" will refer to the next line.

File: Emulator/pal/gc.asm [hunk 3]
```diff
@@ -44,7 +42,7 @@ TTYO,	0						/TTY OUTPUT SUB-ROUTINE.
 		CLA						/CLEAR ACC
 		JMP I TTYO				/RETURN 
 
-MSG,	.						/POINTER TO STRING
+MSG,							/POINTER TO STRING
```
Comment (PontusPih) [inline Emulator/pal/gc.asm]: This was actually a bug. The period "." means address where current line will end up in memory, so in effect the first character in your string is the address of the string.  DOH! I see now that you use AIX so the first character will be skipped... ignore this line :D I wonder if github makes it easy to remove a single line from a pull request.

Anyway. the period is mostly useful to write things like ".+2" which then means the current address +2 locations. It seems popular to write .-. to get zero.

**Comment by PontusPih** [comment]
Notice that there are comments on the commits.

File: Emulator/pal/gc.asm (hunk 2 above)
Comment (sictransit) [reply Emulator/pal/gc.asm]: I agree! However a NOP here won't reserve extra space, just init it to a value != 0.

File: Emulator/pal/gc.asm (hunk 3 above)
Comment (sictransit) [reply Emulator/pal/gc.asm]: Reverted.

**Pull request merged**